packages/web3-eth-accounts/src/index.js

```diff
@@ -468,7 +467,7 @@ function Wallet(accounts) {
 
 Wallet.prototype._findSafeIndex = function(pointer) {
     pointer = pointer || 0;
-    if (_.has(this, pointer)) {
+    if (this != null && this.hasOwnProperty(pointer)) {
```

Comment:
```suggestion
    if (this.hasOwnProperty(pointer)) {
```
You don't need to check if `this` is defined, I'm pretty sure it was always be defined within a function